content/blog/entries/cc-wp-base-theme-docs-content-creation/contents.lr

```diff
@@ -0,0 +1,59 @@
+title: Content Creation Phase: WordPress Base Theme Usage Guide
+---
+categories:
+cc-wp-base-theme
+gsod
+gsod-2020
+---
+author: JackieBinya
+---
+series: gsod-2020-wordpress-base-theme-usage-guide
+---
+pub_date: 2020-11-10
+---
+body:
+For the past couple of weeks we have been actively creating content for the Creative Commons WordPress Base Theme Usage Guide. Currently the draft content is under final review before it is migrated to the main docs site.
+
+## Our Strategy
+Our main goal in creating the docs is to create rich, intuitive, engaging, and beautifully presented community facing documentation for the Creative Commons WordPress Base Theme. 
+
+In alignment with the defined goal, the core focus was to create the docs collaboratively.
+
+The CC WordPress team consists of I, Jacqueline Binya, Hugo Solar and Timid Robot Zehta. Although our team is small it is quite diverse. It consists of a diverse mix of technical skills: I am a junior developer whereas Hugo and Timid are way senior. We also have non-native and native English speakers.
+
+Diversity is important as we hope to create a high quality product that caters for everyone.
+
+My role as the tech writer/frontend developer is to create the content: write the documentation, build the docs site and also to create all illustrative media.
+
+During the content creation phase, the first step involved creating the skeleton of the actual docs site. We created a git branch called _docs_ within the [creative-commons/wp-base-theme](https://docs.google.com/document/d/1XmIsMTLstbhRRSaNFP538YOXJiS0G5QrN6EzuqJfRy4/edit?usp=sharing) repository. All content related to the documentation is persisted in that branch. So,please feel free to contribute. We then used [JamDocs](https://gridsome.org/starters/jamdocs/), a [Gridsome](https://gridsome.org/) theme to quickly scaffold the site. We had to adapt the theme so as to make it meet our own specific needs, this involved overhauling the styles and changing the functionality of some of the features in the theme. After that was completed, we then created a [Google Doc](https://docs.google.com/document/d/1XmIsMTLstbhRRSaNFP538YOXJiS0G5QrN6EzuqJfRy4/edit?usp=sharing) we use for collaboratively writing the draft content for the docs site.
```

Comment:
I think you miss the wp theme base repo link, it's currently linked to a google doc

Reply:
@hugosolar thanks for pointing that out. I have checked all external links in the article and fixed those which were not correct.